examples/cpp/llamacpp/CMakeLists.txt

```diff
@@ -1,20 +1,20 @@
 set(LLAMACPP_SRC_DIR "${torchserve_cpp_SOURCE_DIR}/third-party/llama.cpp")
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
 add_library(llamacpp_handler SHARED src/llamacpp_handler.cc)
 
-set(MY_OBJECT_FILES
-    ${LLAMACPP_SRC_DIR}/ggml.o
-    ${LLAMACPP_SRC_DIR}/llama.o
-    ${LLAMACPP_SRC_DIR}/common.o
-    ${LLAMACPP_SRC_DIR}/ggml-quants.o
-    ${LLAMACPP_SRC_DIR}/ggml-alloc.o
-    ${LLAMACPP_SRC_DIR}/grammar-parser.o
-    ${LLAMACPP_SRC_DIR}/console.o
-    ${LLAMACPP_SRC_DIR}/build-info.o
-    ${LLAMACPP_SRC_DIR}/ggml-backend.o
-
+FetchContent_Declare(
+  llama.cpp
+  GIT_REPOSITORY https://github.com/ggerganov/llama.cpp
+  GIT_TAG b2241
```

Comment:
do we need build llama cpp with -DCMAKE_CXX_FLAGS="-fPIC"?

Reply:
See above